test/functional/test-swipe.js

```diff
@@ -1,404 +0,0 @@
-/**
```

Comment:
Yeah. It was deprecated for a long time. No longer used. Easier to delete now than to refactor.